app/Models/Tracking.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Tracking extends Model
+{
+    protected $fillable = [
+        'user_id',
+        'work_day',
+        'start_time',
+        'end_time',
+        'total_time',
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo(User::class);
```

Comment:
You need `import App\User` to use `User::class` because User not same namespace with this class